pkg/bootstrap/types.go

```diff
@@ -75,6 +75,8 @@ type InfraStatus struct {
 	ControlPlaneTopology   configv1.TopologyMode
 	InfrastructureTopology configv1.TopologyMode
 	InfraName              string
+	// StandaloneManagedCluster is set to true when the cluster is a standalone managed cluster (excl HyperShift)
```

Comment:
nit:
```
// StandaloneManagedCluster is set to true when the cluster is a non-hypershift managed cluster
```
or you could just have
```
// a managed cluster is either a hosted hypershift cluster or standalone
ManagedCluster := infraRes.HostedControlPlane != nil || isStandaloneManagedCluster
```
and use that to check.

Reply:
I don't think that would work, for live migration we support standalone managed clusters but not hypershift.

Reply:
My bad. So ignore the second part